backend/server/issue.go

```diff
@@ -156,78 +31,4 @@ func (s *Server) registerIssueRoutes(g *echo.Group) {
 		}
 		return nil
 	})
-
-	g.PATCH("/issue/:issueID/status", func(c echo.Context) error {
```

Comment:
patch issue status is still used in `BatchIssueActionsV1` by @ecmadao

Reply:
Thanks for the heads-up.